src/device.cpp

```diff
@@ -575,7 +575,9 @@ void cvk_device::build_extension_ils_list() {
 
     // Build list of ILs
     m_ils = {
+#ifdef ENABLE_SPIRV_IL
         MAKE_NAME_VERSION(1, 0, 0, "SPIR-V"),
+#endif
```

Comment:
You also need to stop reporting the cl_khr_il_program extension as SPIR-V is a mandatory IL for implementations that support the extension.